tests/test_legacy.py

```diff
@@ -52,6 +64,18 @@ def test_report_ip__no_api_key_configured(self):
         with self.assertRaises(KeyError):
             abuseipdb.report_ip(ip='192.0.2.123', categories="22")
 
+    def test_report_ip__no_ip_address_provided(self):
+        self.reset_configuration()
+        abuseipdb.configure_api_key('some_API_key')
+        with self.assertRaises(ValueError):
+            abuseipdb.report_ip(categories="22")
+
+    def test_report_ip__no_categories_provided(self):
+        self.reset_configuration()
+        abuseipdb.configure_api_key('some_API_key')
+        with self.assertRaises(ValueError):
+            abuseipdb.report_ip(ip='192.0.2.123')
```

Comment:
Same as before please move any mention of the ip off to a shared or static variable.